edge/pkg/edgehub/controller_test.go

```diff
@@ -66,6 +67,27 @@ func initMocks(t *testing.T) {
 	mockAdapter = edgehub.NewMockAdapter(mockCtrl)
 }
 
+//testYamlGenerator is a structure which is used to generate the test YAML file to test Edgehub config components
+type testYamlGenerator struct {
```

Comment:
can this struct be renamed to something better ? generally a generator generates stuff. we don't have any methods defined on the generator to do so. since this is a wrapper , we can give a more logical name to it.

Reply:
Done, thanks